src/main/java/org/fcrepo/client/FcrepoResponse.java

```diff
@@ -55,6 +72,36 @@ public FcrepoResponse(final URI url, final int statusCode,
     }
 
     /**
+     * {@inheritDoc}
+     * <p>
+     * Implementation note: Invoking this method will close the underlying {@code InputStream} containing the entity
+     * body of the HTTP response.
+     * </p>
+     *
+     * @throws IOException if there is an error closing the underlying HTTP response stream.
+     */
+    @Override
+    public void close() throws IOException {
```

Comment:
Do we want to keep a ref directly to the underlying response so that we can `close` it? It's possible (even if it is not the case now) that so doing would release other resources inside it (besides what we here call `body`).